pkg/provisioner/templates/kubernetes.go

```diff
@@ -147,12 +153,116 @@ echo "you can now access the cluster with:"
 echo "ssh -i <your-private-key> ubuntu@${INSTANCE_ENDPOINT_HOST}"
 `
 
+const NVKindTemplate = `
+
+: ${KIND_K8S_TAG:={{.Version}}}
+
+# The kind image to use. This image will be built if it is not available.
+: ${KIND_IMAGE:="kindest/node:${KIND_K8S_TAG}"}
+
+# The kubernetes tag to build the kind cluster from
+# From https://github.com/kubernetes/kubernetes/tags
+# See also https://hub.docker.com/r/kindest/node/tags
+: ${KIND_K8S_REPO:="https://github.com/kubernetes/kubernetes.git"}
+
+ARCH=$(uname -m)
+if [ "$ARCH" == "x86_64" ]; then
+  ARCH="amd64"
+fi
+if [ "$ARCH" == "aarch64" ]; then
+  ARCH="arm64"
+fi
+
+# Go
+# Set GO version
+GO_VERSION="1.23.6"
+wget https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz -O /tmp/go${GO_VERSION}.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf /tmp/go${GO_VERSION}.linux-amd64.tar.gz
+
+# Add Go to PATH
+if ! grep -q 'export PATH="/usr/local/go/bin:$PATH"' ~/.bashrc; then
+  echo 'export PATH="/usr/local/go/bin:/$HOME/go/bin:$PATH"' >> ~/.bashrc
+fi
+export PATH="/usr/local/go/bin:$HOME/go/bin:$PATH"
+
+# Make
```

Comment:
The comment states that we need `Make`. Do we really need `build-essentials`?

Reply:
done

Reply:
Just so that I'm clear. Is `make` required to run `go install`?